src/main/java/clzzz/helper/logic/parser/pet/AddPetCommandParser.java

```diff
@@ -79,6 +80,11 @@ public AddPetCommand parse(String args) throws ParseException {
         if (argMultimap.getAllValues(PREFIX_DOB).size() > 1) {
             warningMessage += Messages.WARNING_MESSAGE_DOB;
         }
+        if (dateOfBirth.value.isBefore(LocalDate.EPOCH)) {
```

Comment:
I think I can have a pet turtle born before 1970? Actually how do we define a hard lower bound for this?

Reply:
that's why it is a warning not a exception. I set at 1970 cuz I feel it is suitable for most pets